pmtiles/create.md

```diff
@@ -41,6 +48,16 @@ pmtiles convert output.mbtiles output.pmtiles
 
 GDAL has native support for PMTiles starting with version 3.8.0 (2023-11-13), see [gdal.org/drivers/vector/pmtiles](https://gdal.org/drivers/vector/pmtiles.html) for details.
 
+GDAL's [`ogr2ogr`](https://gdal.org/programs/ogr2ogr.html#ogr2ogr) tool supports a wide range of formats as input for creating PMTiles. Below is examples of generating PMTiles from a shape file and based on multiple PostgreSQL/Postgis tables.
+
+```sh
+# Convert shapefile to to pmtiles
+ogr2ogr -dsco MINZOOM=10 -dsco MAXZOOM=20 -f "PMTiles" filename.pmtiles my_shapes.shp
```

Comment:
What client renderer are you using? A MAXZOOM of about 15 or 16 should be the most you ever need for typical GIS datasets, since you can over zoom in the client. that would save a significant amount of space in the final PMTiles.

Reply:
I'm using MapLibre

Will decrease maxzoom next time I'm generating the tiles. 👍